src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -118,81 +99,18 @@
-
-        restAuditingEnabled = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_REST, true);
-        transportAuditingEnabled = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_TRANSPORT, true);
-
-        final List<String> disabledRestCategoriesList = settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_REST_CATEGORIES, DEFAULT_DISABLED_CATEGORIES);
-
-        if (disabledRestCategoriesList.isEmpty() || (disabledRestCategoriesList.size() == 1 && "NONE".equalsIgnoreCase(disabledRestCategoriesList.get(0)))) {
-            disabledRestCategories = EnumSet.noneOf(AuditCategory.class);
-        } else {
-            disabledRestCategories = AuditCategory.parse(disabledRestCategoriesList);
-        }
-
-        if (!disabledRestCategories.isEmpty()) {
-            log.info("Configured categories on rest layer to ignore: {}", disabledRestCategories);
-        }
-
-        final List<String> disabledTransportCategoriesList = settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_TRANSPORT_CATEGORIES, DEFAULT_DISABLED_CATEGORIES);
-
-        if (disabledTransportCategoriesList.isEmpty() || (disabledTransportCategoriesList.size() == 1 && "NONE".equalsIgnoreCase(disabledTransportCategoriesList.get(0)))) {
-            disabledTransportCategories = EnumSet.noneOf(AuditCategory.class);
-        } else {
-            disabledTransportCategories = AuditCategory.parse(disabledTransportCategoriesList);
-        }
-
-        if (!disabledTransportCategories.isEmpty()) {
-            log.info("Configured categories on transport layer to ignore: {}", disabledTransportCategories);
-        }
-
-        logRequestBody = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_LOG_REQUEST_BODY, true);
-        resolveIndices = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_INDICES, true);
-
-        ignoredAuditUsers = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_USERS, defaultIgnoredUsers));
-
-        if(ignoredAuditUsers.size() == 1 && "NONE".equals(ignoredAuditUsers.get(0))) {
-            ignoredAuditUsers.clear();
-        }
-
-        if (ignoredAuditUsers.size() > 0) {
-            log.info("Configured Users to ignore: {}", ignoredAuditUsers);
-        }
-
-        ignoredComplianceUsersForRead = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_IGNORE_USERS, defaultIgnoredUsers));
-
-        if(ignoredComplianceUsersForRead.size() == 1 && "NONE".equals(ignoredComplianceUsersForRead.get(0))) {
-            ignoredComplianceUsersForRead.clear();
-        }
-
-        if (ignoredComplianceUsersForRead.size() > 0) {
-            log.info("Configured Users to ignore for read compliance events: {}", ignoredComplianceUsersForRead);
-        }
+        log.info("Auditing on REST API is {}", auditConfig.isRestApiAuditEnabled() ? "enabled" : "disabled");
```

Comment:
Consider adding excluded REST API audit categories here in case `auditConfig.isRestApiAuditEnabled()` is `true`. The same for Transport API. I don't see them being logged.

Reply:
This was supposed to be done via bulk logging of toString()

Reply:
I will log every param. 
Even is rest or transport api is disabled some categories are excluded. https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java#L718